Modules/Sources/NetworkingCore/Remote/POSRefundsRemoteProtocol.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public protocol POSRefundsRemoteProtocol {
+    func loadRefunds(for siteID: Int64, by orderID: Int64, with refundIDs: [Int64]) async throws -> [Refund]
```

Comment:
Some questions: 
- Since it's a `POSRefundsRemote` protocol,  should this interface return a `[POSRefund]` at this stage?
- Should sit in `NetworkingCore/Remote` as it is now, or in `Network/Remote`? The docs are clearly out of date, so I'm not sure, but by checking the project it would seem that networking core is more for tools than app features? (despite orders and pos orders remote being there as well)